### PR TITLE
Added new ticket mechanism on conference page

### PR DESCRIPTION
**Pull request by AndrewBestbier**
## Motivation

We need to show the statuses of the tickets on the homepage of the conference. This is done in a simple fashion: Showing a button if there are tickets that are available, and showing add to calendar if not.

### CODE REVIEW

File: shared/components/NextConferenceEvent/index.js [hunk 1]
```diff
@@ -1,16 +1,53 @@
 import React, { PropTypes } from 'react';
 import { ExternalLink } from '../ExternalLink';
-
+import { ticketType } from '../../prop-types/ticket-type';
 const locationURL = 'https://goo.gl/maps/GkqTFrJKaUR2';
+const conferenceTicketURL = 'https://ti.to/';
```
Comment (asavin) [inline shared/components/NextConferenceEvent/index.js]: Suppose this one should be editable, or should be changed at some point?

Comment (AndrewBestbier) [inline shared/components/NextConferenceEvent/index.js]: I had the same thought but I don't think we will ever change this link ever once we have it.

Comment (AndrewBestbier) [inline shared/components/NextConferenceEvent/index.js]: I've messaged Amy asking for the link and/or a date when she will get it

Comment (AndrewBestbier) [inline shared/components/NextConferenceEvent/index.js]: Updated the link

**Pull request merged**